### PR TITLE
Add localization support for server messages

### PR DESCRIPTION
**Pull request by def-**
by translating them on client-side.

Texts containing %s/%d are hard this way, replaced some, didn't localize some others. But I guess even incomplete this is better than nothing.

The reason not to do it serverside is because you might have to translate to different language for each player.

### CODE REVIEW

**Comment by fokkonaut** [comment]
Should be done serverside. Server should pop up a voting window when joining a server asking to translate to your language based on IP Address, hitting F3 accepts. Otherwise language should be choosable by /language en/de/ru/tr/idk

**Comment by def-** [comment]
Oh no, I thought about this too simply. It doesn't actually work for replacing parts of texts and doing so will be quite hacky.

**Comment by pure-luck-999** [comment]
Consider using terms not English strings as keys. Like `game.mod.infcroya.class.biologist.title`

Rationale behind this is English strings will be changed too (typos, renames and so on) but you will have to use old string as keys forever.

**Comment by def-** [comment]
Yes, that's clearly better, but I'm not planning to clean up the entire code base in this change :)

**Comment by def-** [comment]
Fix up the server translation support

Still missing:

- scripts/languages needs to create ~~ instead of == if the file is in
  server
- lots of strings with Console()->Print(...) left to Localize

I'm not sure if I like this PR so much anymore. Thoughts?

File: data/languages/german.txt [hunk 1]
```diff
@@ -1213,289 +1213,295 @@ Client message
...
-== hat eine Abstimmung gestartet um dich zu kicken
+~~ hat eine Abstimmung gestartet um dich zu kicken
 
 You can kick only your team member
-== Du kannst nur deine Teammitglieder kicken
+~~ Du kannst nur deine Teammitglieder kicken
 
 Server does not allow voting to move players to spectators
-== Server erlaubt keine Abstimmungen um Spiler zu Beobachtern zu verschieben
+~~ Server erlaubt keine Abstimmungen um Spiler zu Beobachtern zu verschieben
```
Comment (fokkonaut) [inline data/languages/german.txt:1306]: Spiler xd

**Comment by heinrich5991** [comment]
I might take a look at it to do it without sacrificing the ability to change English strings.

**Pull request closed without merging**